With some patches removing c++14 stuff, our gcc-4.9.4 can still
be used to compile MAME. It's about time we get gcc-5.4.0 ready
for Void Linux.

--- scripts/genie.lua	2016-08-31 03:47:01.000000000 +0200
+++ scripts/genie.lua	2016-08-31 07:53:29.475819768 +0200
@@ -970,8 +970,8 @@
 				}
 			end
 		else
-			if (version < 50000) then
-				print("GCC version 5.0 or later needed")
+			if (version < 40900) then
+				print("GCC version 4.9 or later needed")
 				os.exit(-1)
 			end
 				buildoptions {
